src/pivot-table/contexts/StyleProvider.tsx

```diff
@@ -12,8 +14,18 @@ const StyleContext = createContext<StyleService>(NOOP_STYLE_SERVICE);
 
 export const useStyleContext = (): StyleService => useContext(StyleContext);
 
-const StyleProvider = ({ children, styleService }: StyleProviderProps): JSX.Element => (
-  <StyleContext.Provider value={styleService}>{children}</StyleContext.Provider>
-);
+const StyleProvider = ({ children, styleService, layoutService }: StyleProviderProps): JSX.Element => {
```

Comment:
I do think we should expose cell heigth via the style provider. Not really sure about the "style service" and mixin it with props resolved here.

But once we support more styling options from the styling panel we should revisit this and figure a good way forward. Ex. is the any value in keeping the style service?